### PR TITLE
VACMS-10262 refactor wysiwyg and rich text char limit 1000

### PR DESCRIPTION
**Pull request by tjheffner**
## Description
Closes #10262 https://github.com/department-of-veterans-affairs/va.gov-cms/issues/10262

## Testing done
Refactored both, removed the rich text entirely. It was the same structure as wysiwyg, just typed the formatter to accept either paragraph type, and pointed both queries at the same formatter. Seems to work just fine.
## Screenshots
rich text char limit 1000:
Screen Shot 2022-09-29 at 11 12 49 AM

wysiwyg:
Screen Shot 2022-09-29 at 11 15 30 AM

the two "asdfasdf" divs do appear in the json output here.. someone testing a q&a paragraph it looks like. not completely sure, didn't dig further than that
https://content-build-medc0xjkxm4jmpzxl3tfbcs7qcddsivh.ci.cms.va.gov/jsonapi/paragraph/wysiwyg

## QA steps
Pull branch, copy these demo pages with queries.formatData() instead of the generalEntityDataService

`pages/demo/richText/index.tsx`:
```js
import { drupalClient } from '@/lib/utils/drupalClient'
import { GetStaticPropsContext, GetStaticPropsResult } from 'next'
import { DrupalJsonApiParams } from 'drupal-jsonapi-params'
import Container from '@/templates/common/container'
import {
  ParagraphRichTextCharLimit1000,
  ParagraphResourceType,
} from '@/types/dataTypes/drupal/paragraph'
import { Wysiwyg } from '@/templates/components/wysiwyg'
import { queries } from '@/data/queries'

interface RichTextCharLimit1000PageProps {
  richTextCharLimit1000CollectionProps: any
  className: string
}

const RichTextCharLimit1000Page = ({
  richTextCharLimit1000CollectionProps,
  className,
}) => {
  if (!richTextCharLimit1000CollectionProps)
    richTextCharLimit1000CollectionProps = []

  return (
    <Container className="container">
      {richTextCharLimit1000CollectionProps.map((richTextCharLimit1000) => (
        <Wysiwyg
          key={richTextCharLimit1000.id}
          {...richTextCharLimit1000}
          className={className}
        />
      ))}
    </Container>
  )
}

export default RichTextCharLimit1000Page

export async function getStaticProps(
  context: GetStaticPropsContext
): Promise<GetStaticPropsResult<RichTextCharLimit1000PageProps>> {
  const params = new DrupalJsonApiParams()
  params.addPageLimit(20)

  const richTextCharLimit1000Collection =
    await drupalClient.getResourceCollectionFromContext<
      ParagraphRichTextCharLimit1000[]
    >(ParagraphResourceType.RichTextCharLimit1000, context, {
      params: params.getQueryObject(),
    })

  const richTextCharLimit1000CollectionProps = richTextCharLimit1000Collection.map(richText => queries.formatData('paragraph--rich_text_char_limit_1000', richText))

  return {
    props: {
      richTextCharLimit1000CollectionProps,
      className: 'processed-content',
    },
  }
}
```

`pages/demo/wysiwyg/index.tsx`:
```js
import { drupalClient } from '@/lib/utils/drupalClient'
import { GetStaticPropsContext, GetStaticPropsResult } from 'next'
import { DrupalJsonApiParams } from 'drupal-jsonapi-params'
import Container from '@/templates/common/container'
import {
  ParagraphResourceType,
  ParagraphWysiwyg,
} from '@/types/dataTypes/drupal/paragraph'
import { Wysiwyg } from '@/templates/components/wysiwyg'
import { queries } from '@/data/queries'

interface WysiwygPageProps {
  wysiwygCollectionProps: any
  className: string
}

const WysiwygPage = ({
  wysiwygCollectionProps,
  className,
}: WysiwygPageProps) => {
  return (
    <Container className="container">
      {wysiwygCollectionProps.map((wysiwygProps) => (
        <Wysiwyg key={wysiwygProps.id} {...wysiwygProps} />
      ))}
    </Container>
  )
}

export default WysiwygPage

export async function getStaticProps(
  context: GetStaticPropsContext
): Promise<GetStaticPropsResult<WysiwygPageProps>> {
  const params = new DrupalJsonApiParams()
  params.addPageLimit(20)

  const wysiwygCollection = await drupalClient.getResourceCollectionFromContext<
    ParagraphWysiwyg[]
  >(ParagraphResourceType.Wysiwyg, context, {
    params: params.getQueryObject(),
  })
  const wysiwygCollectionProps = wysiwygCollection.map(wysiwyg => queries.formatData('paragraph--wysiwyg', wysiwyg))

  return {
    props: {
      wysiwygCollectionProps,
      className: 'processed-content',
    },
  }
}
```

visit http://localhost:3000/demo/wysiwyg/ and http://localhost:3000/demo/richText/ and verify that content is rendering as expected

### Definition of Done

- [x] Documentation has been updated, if applicable.
- [x] Tests have been added if necessary.
- [x] Automated tests have passed.
- [x] Code Quality Tests have passed.
- [x] Acceptance Criteria in related issue are met.
- [x] Manual Code Review Approved.

### CODE REVIEW

File: src/data/delegators/entityMetaProvider.tsx [hunk 1]
```diff
@@ -1,6 +1,4 @@
 import { DrupalJsonApiParams } from 'drupal-jsonapi-params'
```
Comment (tjheffner) [inline src/data/delegators/entityMetaProvider.tsx:1]: we may be able to nuke this entirely now?

File: src/data/queries/index.ts [hunk 2]
```diff
@@ -13,6 +13,7 @@ import * as AudienceTopics from './audienceTopics'
 import * as Alert from './alert'
```
Comment (tjheffner) [inline src/data/queries/index.ts:13]: just noting i alphabetized the paragraph queries in addition to adding the two for wysiwyg

File: src/data/delegators/entityMetaProvider.tsx (hunk 1 above)
Comment (timcosgrove) [reply src/data/delegators/entityMetaProvider.tsx:1]: Indeed, I'll open a quick branch to do that.

File: src/data/queries/index.ts (hunk 2 above)
Comment (timcosgrove) [reply src/data/queries/index.ts:13]: Thank you!

**Review by timcosgrove — APPROVED** [review]

**Pull request merged**